src/cli-options.ts

```diff
@@ -146,6 +146,31 @@ Example:
     Saving partially upgraded package.json
 `
 
+/** Extended help for the filterResults option. */
+const extendedHelpFilterResults = (): string => {
+  return `Filters the results of update based on user provided function. Only available in .ncurc.js or when importing npm-check-updates as a module:
+
+    ${chalk.gray(`/**
+      @param packageName                   The name of the dependency.
+      @param currentVersion                Current version declaration (may be range)
+      @param currentVersionSemver          Current version declaration in semantic versioning format (may be range)
+      @param upgradedVersion               Upgraded version declaration (may be range)
+      @param upgradedVersionSemver         Upgraded version declaration in semantic versioning format (may be range)
+      @returns                             A result in boolean format - true or false.
```

Comment:
"Return true if the upgrade should be kept, otherwise it will be ignored."

Reply:
fixed 🚀